### ISSUE TITLE
PureScript port: next steps

### ISSUE THREAD

**Issue by hdgarrood** [purescript-contrib/pulp#119]
As far as I can tell, the code in the `purescript` branch now does everything that the `master` branch does. I'm currently trying to run the integration tests on a windows VM, I'll let you know how that goes.

Oh, also, I don't know how webpack works; `pulp server` seems to work but it would be good to get someone who actually uses it to check.

There are a few more integration tests now in the purescript branch too, see: https://travis-ci.org/bodil/pulp/jobs/94196594#L2416

I haven't ported the other tests, but I think the integration tests are enough.

Anyway, what do you think we should do now?

**Comment by hdgarrood** [comment]
Oh wait, I've just realised `server` doesn't work at the moment, and the `--monochrome` option is also not implemented. Will get on those when I have an opportunity.

**Comment by hdgarrood** [comment]
Ok, those two things are done.

**Comment by bodil** [comment]
So... are you confident about making a release?

**Comment by hdgarrood** [comment]
I'd rather test a bit more thoroughly first. It would be really good to get someone who actually uses `server` to help test, for example (I still don't really understand webpack). I would be confident enough to release a release candidate version, though, like `5.1.0.rc-1` or something.

**Comment by hdgarrood** [comment]
Oh wait, I also need to tell it to get the version from `package.json`. One sec.

**Comment by bodil** [comment]
I'll try and review the `pulp server` bit.

**Comment by hdgarrood** [comment]
It's now getting the version from `package.json`.

I tried to run the tests on a Windows VM but they're nearly all failing because the bower file it expects to be executable isn't. Almost all the tests start by doing `pulp init`, so this makes them all fail.

I found another issue, which might be related, which is that `pulp build` seems to hang instead of exiting if it can't find `bower.json`.

These issues seem to be present on pulp 5.0.2 as well. I'll try to see if I can fix them.

**Comment by bodil** [comment]
The Windows issue is probably #108.

**Comment by hdgarrood** [comment]
Ok - #108 is now fixed in the purescript branch, and so is #121. Also, the integration tests now run on Windows. I'm currently looking at appveyor, which seems to basically be like Travis CI but for Windows, but I think that's just a nice-to-have.

So the only thing now that makes me hesitant about releasing is `pulp server`. But I am more confident about that now that I've tested it a bit more thoroughly. Also given that there are a few issues that are fixed in the purescript port but not in the javascript version (the ones I've mentioned, and also #111), I'm more inclined to release.
